src/daemon/daemon.cpp

```diff
@@ -124,7 +124,7 @@ bool t_daemon::run(bool interactive)
       return false;
     mp_internals->rpc.run();
 
-    daemonize::t_command_server* rpc_commands;
+    daemonize::t_command_server* rpc_commands = NULL;
```

Comment:
Can you update this to a `std::unique_ptr<daemonize::t_command_server>` instead? The pointer will always be initialized with `unique_ptr`, and it fixes a memory leak here.